### ISSUE TITLE
Output spam - No message entry

### ISSUE THREAD

**Issue by fnoop** [01org/mavlink-router#25]
```
Feb 23 23:58:39 fnoop-raspberry mavlinkrouter.sh[9455]: (endpoint.cpp:269) No message entry for 178
Feb 23 23:58:39 fnoop-raspberry mavlinkrouter.sh[9455]: (endpoint.cpp:269) No message entry for 182
Feb 23 23:58:39 fnoop-raspberry mavlinkrouter.sh[9455]: (endpoint.cpp:269) No message entry for 163
Feb 23 23:58:39 fnoop-raspberry mavlinkrouter.sh[9455]: (endpoint.cpp:269) No message entry for 165
Feb 23 23:58:39 fnoop-raspberry mavlinkrouter.sh[9455]: (endpoint.cpp:269) No message entry for 173
Feb 23 23:58:39 fnoop-raspberry mavlinkrouter.sh[9455]: (endpoint.cpp:269) No message entry for 193
Feb 23 23:58:39 fnoop-raspberry mavlinkrouter.sh[9455]: (endpoint.cpp:269) No message entry for 178
Feb 23 23:58:39 fnoop-raspberry mavlinkrouter.sh[9455]: (endpoint.cpp:269) No message entry for 182
```

Looks like is when mavlink-router can't look up the incoming message against mavlink library?  Not sure why this is the case, but the output should at least be an option or rate-limited, this is killing the system journal logging in this case, as it's launched from systemd.

**Comment by lucasdemarchi** [comment]
Yep... this message was recently added and should be fixed.

For the unknown messages, it's probably related to using common.xml to generate the headers. You are using ardupilot, right?  We need to have an option to use another xml.

**Comment by fnoop** [comment]
Yes exactly, using ardupilot (copter 3.5-dev/master).

**Comment by lucasdemarchi** [comment]
@fnoop the excessive logging is being fixed by #27. We will add another patch to allow to use the ardupilot xml instead of common.xml.

**Comment by fnoop** [comment]
Fixed, thanks very much :)))
